web/src/contexts/governmentContractingContext.ts

```diff
@@ -0,0 +1,55 @@
+import {
```

Comment:
note: delete this as we are not using it anymore

Reply:
done